src/test/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/controllers/OcrValidationControllerTest.java

```diff
@@ -160,7 +160,7 @@ void should_return_form_not_found_exception_when_form_type_case_does_not_match()
         MvcResult mvcResult = mockMvc
             .perform(
                 post("/forms/Personal/validate-ocr") //only PERSONAL is valid form type
-                    .contentType(MediaType.APPLICATION_JSON_VALUE)
+                    .contentType(APPLICATION_JSON_VALUE)
```

Comment:
are changes is this file related to dependencies update?

Reply:
Yes. Usage of `APPLICATION_JSON_UTF8` is deprecated in Spring Boot 2.2. Only `APPLICATION_JSON_VALUE` is relevant to this class so moving to static import instead